### DISCUSSION TITLE
Do scheduled workflows run from other branches?

### DISCUSSION THREAD

**Discussion by VladimirSlavik** [community/community#36573] [Actions]
I have a workflow that runs `on: schedule - cron: (...)`. Will this workflow trigger from non-default branches? Or rather - if I have it on the default branch and on another one, will it trigger twice?

And, while I'm at it... How would I learn that from the docs? I mean [this](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule) does not tell me "it runs on all branches" or "it runs only on default branch" or anything of that kind... (Nor does [that](https://docs.github.com/en/actions/using-workflows/triggering-a-workflow), fwiw.)

**Comment by AF250329** [comment]
Hi

I am not sure where it is written in documentation but all the workflows I can use must exist in `main` branch. Otherwise there are not visible in `Actions` menu and could not be triggered. Only workflows in `main` branch are considered and used.

Could you check it with `on: schedule: ` event ? Just create temporary branch with this kinda workflow and check if it running at all in it branch ? I think minimum time for scheduler it is 5 min - so you won't wait long for it

**Reply by VladimirSlavik** [reply]
Thanks for the reply. So I tried that, and it appears that scheduled workflows run only from the main branch.

Also, surprise, the scheduler picks up workflow changes too slow, so you have to wait far more than 5 minutes. In my experience it was more like an hour, the workflow simply did not run when scheduled earlier. Also do not forget that github runners are on UTC...
